data-products/src/main/scala/org/sunbird/analytics/util/RedisCacheUtil.scala

```diff
@@ -0,0 +1,53 @@
+package org.sunbird.analytics.util
+
+import java.time.Duration
+
+import org.ekstep.analytics.framework.conf.AppConf
+import org.ekstep.analytics.framework.util.{JSONUtils, JobLogger}
+import redis.clients.jedis.{Jedis, JedisPool, JedisPoolConfig}
+
+
+class RedisCacheUtil {
```

Comment:
Why is this class added?

Reply:
This is used to mock user information from redis